yarn-project/end-to-end/bootstrap.sh

```diff
@@ -74,7 +74,6 @@ function test_cmds {
   echo "$prefix simple e2e_nested_contract/manual_public"
 
   echo "$prefix simple e2e_nft"
-  echo "$prefix simple e2e_non_contract_account"
```

Comment:
This test was just weird - it contains just one test case which looks useless at this point and the test case has nothing to do with non_contract_accounts. So I just nuked it all.